Octokit.Reactive/Clients/ObservableRepositoriesClient.cs

```diff
@@ -429,7 +535,29 @@ public IObservable<RepositoryContributor> GetAllContributors(string owner, strin
             if (includeAnonymous)
                 parameters.Add("anon", "1");
 
-            return _connection.GetAndFlattenAllPages<RepositoryContributor>(endpoint, parameters);
+            return _connection.GetAndFlattenAllPages<RepositoryContributor>(endpoint, parameters, options);
```

Comment:
oh dear is this another place we missed passnig the `ApiOptions` through, on the previous changes?  nice pickup